integration-tests/environments/react-native/ios/RealmReactNativeTests/AppDelegate.m

```diff
@@ -53,7 +53,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 {
 #if DEBUG
-  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
+  return [NSURL URLWithString:[[[[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil] absoluteString] stringByAppendingString:@"&inlineSourceMap=true" ]];
```

Comment:
I changed this to make it easier to debug JavaScript with Safari.

Reply:
It makes sense since we are going to make this modification of a regular basis.

Reply:
A comment about source map and Safari might be appropriate.